client/state/data-layer/wpcom/concierge/shifts/index.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { noRetry } from 'state/data-layer/wpcom-http/pipeline/retry-on-failure/policies';
+import { updateConciergeShifts } from 'state/concierge/actions';
+import { errorNotice } from 'state/notices/actions';
+import { CONCIERGE_SHIFTS_FETCH } from 'state/action-types';
+
+export const requestFetchConciergeShifts = ( { dispatch }, action ) => {
+	const { scheduleId } = action;
+
+	dispatch(
+		http(
+			{
+				method: 'GET',
+				path: `/concierge/schedules/${ scheduleId }/shifts`,
+				apiNamespace: 'wpcom/v2',
+				retryPolicy: noRetry(),
```

Comment:
Any reason for not retrying? If so, it should probably be well-documented here.

Reply:
I didn't find uses of the `exponentialBackoff` policy, so I thought it was a work-in-progress. It is using `exponentialBackoff` now after b71a08a.

Reply:
Actually the exponential backoff policy is the default - there's no need to provide any `retryPolicy` unless we want to intentionally change it from the default.

Reply:
Somehow I missed it as the default. It is now removed by 8c403b0.